autorest/utility_test.go

```diff
@@ -239,6 +240,36 @@ func TestEncodeWithValidNotPathQuery(t *testing.T) {
 	}
 }
 
+func TestMapToValues(t *testing.T) {
+	m := map[string]interface{}{
+		"a": "a",
+		"b": 2,
+	}
+	v := url.Values{
+		"a": []string{"a"},
+		"b": []string{"2"},
+	}
+	if !reflect.DeepEqual(MapToValues(m), v) {
+		t.Fatalf("autorest: MapToValues method failed to return correct values - expected(%v) got(%v)", v, MapToValues(m))
+	}
+}
+
+func TestMapToValuesWithArrayValues(t *testing.T) {
```

Comment:
I'm not sure if this test is guaranteed to pass tomorrow. Can you please validate if url.Values.Add preserves the order in which you add multiple values to the same key?